ruby.json

```diff
@@ -1,6 +1,6 @@
 {
     "id" : "ruby",
-    "version" : "0.5.4",
+    "version" : "0.5.5",
```

Comment:
I guess this can be a 0.6.0 version.

Reply:
Yeah, I'll change these versions when the FTs finish successfully.

IIRC after the version is changed the FTs will fail due to the pessimistic version range in https://github.com/getgauge/gauge-tests/blob/a01faf42b3e2f8e295ae3b7f31ce43768b8bc684/resources/LocalTemplates/ruby/Gemfile#L5-L7.

Probably should either get the version from an env var, or install whatever version is on the path/branch, if possible, or maybe just use a `>= 0.5.3, < 1.0.0` in the FT install constraint to work around semver rules.

Reply:
Have loosened this version range upstream now to avoid this chicken vs. egg problem on making semver incompatible Gem bumps.